recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -14,5 +14,10 @@ conda install -n root --yes --quiet jinja2 conda-build anaconda-client
 :: KLUDGE to work around changes in conda-build 2.0.0
 conda install -n root --yes --quiet conda-build=1
 
+:: Needed for building extensions in python2.7 x64 with cmake.
+:: Since python version and arch is not known at this point, install it everywhere.
+conda install -n root --yes --quiet vs2008_express_vc_python_patch
+call setup_x64
```

Comment:
Yes, `conda-forge-build-setup` is bootstrapped actually. Can see the results in the [previous build]( https://ci.appveyor.com/project/conda-forge/conda-forge-build-setup-feedstock/build/1.0.141/job/38uuayok0720nnb9#L159 ). Details for how the bootstrapping works can be found in PR ( https://github.com/conda-forge/conda-smithy/pull/285 ) and are implemented for this feedstock in PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/43 ).